### ISSUE TITLE
Consider adding ClojureScript localization utils

### ISSUE THREAD

**Issue by ptaoussanis** [taoensso/tower#53]
The ClojureScript code currently only provides translation facilities. Input, suggestions, and PRs (especially) welcome on this.

Cheers! :-)

**Comment by bensu** [comment]
I'd love to help with the ClojureScript side of things. Is it okay if I start by porting cljx to cljc?

**Comment by ptaoussanis** [comment]
Hey Sebastian,

Just to clarify: you have in mind some way of adding support for JS-side localization stuff (`fmt`, `parse`, etc.)? I'd be very open to that (!), but would suggest we discuss details first (approach, etc.).

In case it's relevant, I should caution that I'll be replacing the entire translations design in a future pending update.

> Is it okay if I start by porting cljx to cljc?

cljc requires Clojure 1.7+ which is still pretty new. I've been trying to target Clojure 1.5+ recently (was actually 1.4+ until the release of 1.7). Requiring cljc would potentially break a lot of users, so maybe something best avoided when possible?

**Comment by bensu** [comment]
Yes, there are bits of `fmt` and `parse` that I'll need to use from ClojureScript, and I'm planning to write wrappers around Google Closure's i18n facilities. To do so, I'll rather work with cljc and the latest ClojureScript. If that is a problem, I'll work on my own following the tower api in a separate project and report back as soon as I have something that can kickstart a discussion.

**Comment by ptaoussanis** [comment]
> I'll work on my own following the tower api in a separate project and report back as soon as I have something that can kickstart a discussion.

Sounds good, feel free to ping if you need anything I can assist with! Cheers :-)
